### PR TITLE
Add a code of conduct for the project

### PR DESCRIPTION
**Pull request by lornajane**
# About this change - What it does

Add the code of conduct (https://www.contributor-covenant.org/ v2.1) with Aiven's contact details to the project. Added the badge to the readme (and updated the copyright year while I was there)

I suggest we review and discuss here, but leave open for any community comments or feedback for a couple of weeks before merging.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -7,7 +7,12 @@ An `open-source <https://github.com/aiven/karapace/blob/master/LICENSE>`_ implem
 of `Kafka REST <https://docs.confluent.io/platform/current/kafka-rest/index.html#features>`_ and
 `Schema Registry <https://docs.confluent.io/platform/current/schema-registry/index.html>`_.
 
-.. image:: https://github.com/aiven/karapace/actions/workflows/tests.yml/badge.svg
+|Tests| |Contributor Covenant|
+
+.. |Tests| image:: https://github.com/aiven/karapace/actions/workflows/tests.yml/badge.svg
+
+.. |Contributor Covenant| image:: https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg
```
Comment (jlprat) [inline README.rst:14]: nit pick: Name this "Contributor Covenant Code of Conduct" or "Contributor Covenant CoC"?

**Review by jlprat — APPROVED** [review]
I fully support the use of the Contributor Covenant Code of Conduct.
I'm also fine with the use of `opensource@aiven.io` as the e-mail to contact for this reason.

**Comment by jlprat** [comment]
BTW, the linter complaints about a missing empty line at EOF.

**Review by tvainika — APPROVED** [review]
LGTM, but linter failure needs action.

**Review by jlprat — APPROVED** [review]

**Pull request merged**